modules/rn_3_13_0.adoc

```diff
@@ -55,21 +55,26 @@ The following enhancements have been made to the {productname} v2 UI.
 
 The following configuration fields have been added to {productname} {producty}.
 
-[id="oauth-reassign-configuration-field"]
-=== OAuth access token reassignment configuration field 
+[id="disable-pushes-configuration-field"]
+=== Disabling pushes to the {productname} registry configuration field
+
+In some cases, a read-only option for {productname} is not possible since it requires inserting a service key and other manual configuration changes. With the release of {productname} 3.13, a new configuration field has been added: `DISABLE_PUSHES`. 
+
+When `DISABLE_PUSHES` is set to `true`, users are unable to push _new_ images to the registry; however, if the registry already contains the image and a new tag is added, the operation succeeds. All other registry operations continue as normal, for example, changing tags, editing a repository, robot account creation and deletion, user creation, and so on.
```

Comment:
Steven, pls review this chat. My understand is: "when DISABLE_PUSHES: true, can't push image even the registry already contains the image"
https://redhat-internal.slack.com/archives/G7VFPAY7Q/p1726801217423289